### PR TITLE
Make noarch

### LINKED ISSUE #57: Py310 version blocked due to upstream requierements
https://github.com/docker/docker-py/pull/2857

### PR DESCRIPTION
**Pull request by BastianZim**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #62 
Closes #57

This PR
1. deletes the patches
2. Updates to v6
3. Makes it noarch
4. Adds Grayskull to the auto merge so new dependencies are added

### CODE REVIEW

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -58,14 +38,19 @@ test:
     - docker.utils.ports
     - docker.transport.unixconn
   commands:
-    - python -c "import docker; docker.DockerClient.from_env(version='auto')"  # [win]
-    - python -c "import pkg_resources; pkg_resources.require('docker')"  # [win]
+    - pip check
+    - python -c "import docker; docker.DockerClient.from_env(version='auto')"
+    - python -c "import pkg_resources; pkg_resources.require('docker')"
```
Comment (BastianZim) [inline recipe/meta.yaml:43]: This was in the recipe, not sure if it is needed.

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipe/meta.yaml:43]: That is a flaky test anyway. I'm ok removing it.

Comment (BastianZim) [reply recipe/meta.yaml]: Removed :)

**Comment by BastianZim** [comment]
@ocefpaf Good to merge. :)

**Comment by BastianZim** [comment]
Thank you :)

**Comment by ocefpaf** [comment]
Thank you for this! The more pkgs we can make as noarch the better!

**Comment by maresb** [comment]
`bot.inspection=update-grayskull`??? Since when did this become a thing? :grin: 

Is it documented anywhere? I'm very curious.

**Comment by BastianZim** [comment]
Since a couple of months and is _seriously_ helpful. 

It's not yet documented because it's still in beta but you can use it. 

Just add the line like here https://github.com/conda-forge/amazon-braket-sdk-feedstock/blob/9bf8e271b9845245b66a3b191b7af844877b84da/conda-forge.yml#L5

And Grayskull will update your PRs like this: https://github.com/conda-forge/amazon-braket-sdk-feedstock/pull/58

It doesn't delete packages and only works on python packages where Grayskull would work "standalone" but for most pure pypi packages it's incredibly helpful.

**Comment by maresb** [comment]
Thanks for the overview! I will definitely play around with that.

But suppose I want to test it on a feedstock without a release. Thus I open a fresh PR where I set `update-grayskull`. Is there any way I can summon regro-cf-autotick-bot within my PR in order to generate a commit so I can see what it would change?

**Comment by BastianZim** [comment]
Hmm no I don't think so. The only way is currently to add it to a feedstock and wait for a new release. 

If you just want to see what it generates, I guess you can just run Grayskull normally?

**Comment by BastianZim** [comment]
This was the PR by the way for it https://github.com/regro/cf-scripts/pull/1503

**Comment by maresb** [comment]
> This was the PR by the way for it https://github.com/regro/cf-scripts/pull/1503

Ooh, yes, this is what I was looking for, thanks!

**Comment by ocefpaf** [comment]
Multiple outputs would be quite hard to support and I would not invest time in that at the moment. Selectors on the other hand, we could implement something that is opinionated towards what we do in conda-forge. Grayskull kinds of already does that if one uses the `--strict-conda-forge` option. Removing deps is the low hanging fruit here though.

**Comment by BastianZim** [comment]
> Multiple outputs would be quite hard to support and I would not invest time in that at the moment.

Multiple outputs actually already work in Grayskull, they're just not implemented in the action, I think (And, to be honest, I haven't gotten them to work on a package yet myself, it never finds them). Ref: https://github.com/conda-incubator/grayskull/pull/341

> Selectors on the other hand, we could implement something that is opinionated towards what we do in conda-forge. 

Selectors should already work, and they do from what I can see. Just not 100% of the time. So if you're relying on Grayskull for auto merge, that might not always work. Reason I mentioned them above is more that I wanted to describe the situation in which Grayskull works 100%. Ref: https://github.com/conda-incubator/grayskull/blob/8bcb3d78cfc8b647c65b4ea93406c681bb068684/grayskull/strategy/config.yaml#L460

> Removing deps is the low hanging fruit here though.

Yes true. It was excluded in the initial PR so I guess there is a reason but not sure.

**Comment by ocefpaf** [comment]
> Multiple outputs actually already work in Grayskull, they're just not implemented in the action,

That is great. I think I saw that PR but for some reason forgot about it.

> Reason I mentioned them above is more that I wanted to describe the situation in which Grayskull works 100%

Got it.

> Yes true. It was excluded in the initial PR so I guess there is a reason but not sure.

Or no reason. Just a bug we need to address :smile:

**Comment by BastianZim** [comment]
That could also be the case 😄

**Pull request merged**